ios/Classes/Error/BleError.h

```diff
@@ -1,5 +1,5 @@
 #import "BleErrorCode.h"
-#import "CommonTypes.h"
+#import "CommonTypes1.h"
```

Comment:
BlemulatorCommonTypes?

Reply:
Changed according to the review suggestion.